perllib/FixMyStreet/Cobrand/Thamesmead.pm

```diff
@@ -115,4 +115,60 @@ sub geocoder_munge_results {
 
 }
 
+sub report_new_is_on_thamesmead_area {
+
+    my ( $self ) = @_;
+
+    my ($x, $y) = Utils::convert_latlon_to_en(
+        $self->{c}->stash->{latitude},
+        $self->{c}->stash->{longitude},
+        'G'
+    );
+
+    for my $typename ('hardsurfaces', 'grass', 'water', 'treegroups', 'planting') {
+
+        my $cfg = {
+            url => "https://tilma.mysociety.org/mapserver/thamesmead",
+            srsname => "urn:ogc:def:crs:EPSG::27700",
+            typename => $typename,
+            filter => "<Filter><Contains><PropertyName>Extent</PropertyName><gml:Point><gml:coordinates>$x,$y</gml:coordinates></gml:Point></Contains></Filter>",
+            outputformat => "GML3",
+        };
+
+        my $features = FixMyStreet::Cobrand::UKCouncils->new->_fetch_features($cfg, $x, $y);
```

Comment:
So this is going to make 5 external Tilma calls every time somewhere in Bexley or Greenwich is clicked? That doesn't sound ideal. Can we not make one request with all the typenames at once and see which one we get back?

Reply:
I've got this making the one call (thanks for the help - I got the brackets wrong and then thought it an unlikely solution anyway so was trying other stuff). Problem at the moment is that $features may now be an array ref as planters/trees seem to occupy the same/similar locations

Reply:
I'm going to make the assumption that planters/beds are a subset of trees, looks like it from e.g.
<img src="https://user-images.githubusercontent.com/154364/187965554-34dd9c97-3dc9-48d7-9b09-8b5a6a359828.png" float="left"> vs <img src="https://user-images.githubusercontent.com/154364/187965593-26516bb4-fb87-40aa-8c0b-732c02abcbca.png" float="left">